main.test.js

```diff
@@ -0,0 +1,94 @@
+const request = require("supertest");
+const app = require("./app");
+const fsPromise = require("fs/promises");
+const { DB } = require("./api/shorturl");
+const { response } = require("express");
+const { locals } = require("./app");
+const testsUrl = { url: "https://www.youtube.com/" };
+const expectedResponse = {
+  fullUrl: "https://www.youtube.com/",
+  shortid: "gv5TqP1iQ",
+  createdAt: "5.3.2021, 12:59:24",
+  clicks: 1,
+};
+const newUrl = { url: "https://finviz.com/" };
+
+/********resetting the tests database after each test to it`s initial value*******/
+afterAll(async () => {
+  await fsPromise.writeFile(
+    `./DataBase/testDataBase.json`,
+    JSON.stringify([expectedResponse], null, 4),
+    (e) => {
+      console.log(e);
+    }
+  );
+});
+/******************************Testing POST method*******************************************/
+
+describe("POST route tests", () => {
+  it("Should save to the dataBase new given url's", async () => {
+    const DBLengthBefore = DB.urls.length;
+    const res = await request(app).post("/api/shorturl/new").send(newUrl);
+
+    expect(res.status).toBe(201);
+    expect(DB.urls.length).toBeGreaterThan(DBLengthBefore);
```

Comment:
I would check for an exact equality, to make sure only one (or 2, or 3, whatever you expect) entries were added.